### DISCUSSION TITLE
复制（Replicant）节点会同步核心（Core）节点的消息内容吗？

### DISCUSSION THREAD

**Discussion by onejimmyboy** [emqx/emqx-operator#975] [General]
复制（Replicant）节点会同步核心（Core）节点的消息内容吗？
如果是集群模式，三个core节点，宕机几个的话，消息是不是不会丢，在其他core节点也会有存储？

**Comment by Rory-Z** [comment]
This repo is focus to deploy EMQX on K8s,

Maybe you can create discussion to [emqx/emqx.git](https://github.com/emqx/emqx) or [ask EMQX](https://askemq.com/c/emqx/5).
